tagstudio/resources/translations/en.json

```diff
@@ -0,0 +1,144 @@
+  "library.volume": "Volume",
+  "library.anthology": "Anthology",
+  "library.magazine": "Magazine",
+  "library.publisher": "Publisher",
+  "library.guest_artist": "Guest Artist",
+  "library.composer": "Composer",
+  "library.comments": "Comments",
+  "open_library.no_tagstudio_library_found": "No existing TagStudio library found at '%{path}'. Creating one.",
+  "open_library.library_creation_return_code": "Library Creation Return Code:",
+  "open_library.title": "Library",
+  "dialog.open_create_library": "Open/Create Library",
+  "splash.open_library": "Opening Library",
+  "status.save_success": "Library Saved and Closed!",
+  "status.backup_success": "Library Backup Saved at:",
+  "status.search_library_query": "Searching Library for",
+  "status.enumerate_query": "Query:%{query}, Frame: %{i},  Length: %{len(f)}",
+  "status.number_results_found": "%{len(all_items)} Results Found for \"%{query}\" (%{format_timespan(end_time - start_time)})",
+  "status.results_found": "Results",
+  "dialog.save_library": "Save Library",
+  "dialog.refresh_directories": "Refreshing Directories",
+  "dialog.scan_directories": "Scanning Directories for New Files...\nPreparing...",
+  "dialog.scan_directories.new_files": "Scanning Directories for New Files...\n%{x + 1} File%{\"s\" if x + 1 != 1 else \"\"} Searched, %{len(self.lib.files_not_in_library)} New Files Found",
+  "tooltip.open_library": "Ctrl+O",
+  "tooltip.save_library": "Ctrl+S",
+  "progression.running_macros.new_entries": "Running Macros on New Entries",
+  "progression.running_macros.one_new_entry": "Running Configured Macros on 1/%{len(new_ids)} New Entries",
+  "progression.running_macros.several_new_entry": "Running Configured Macros on %{x + 1}/%{len(new_ids)} New Entries",
+  "file_opener.open_file": "Opening file:}",
+  "file_opener.not_found": "File not found:",
+  "file_opener.command_not_found": "Could not find %{command_name} on system PATH",
+  "add_field.add": "Add Field",
+  "generic.remove_field": "Remove Field",
+  "generic.file_extension": "File Extensions",
+  "generic.open_file": "Open file",
+  "generic.open_file_explorer": "Open file in explorer",
+  "generic.cancel": "Cancel",
+  "generic.add": "Add",
+  "generic.name": "Name",
+  "generic.shorthand": "Shorthand",
+  "generic.aliases": "Aliases",
+  "generic.color": "Color",
+  "generic.delete": "Delete",
+  "generic.exclude": "Exclude",
+  "generic.include": "Include",
+  "generic.done": "Done",
+  "generic.open_all": "Open All",
+  "generic.close_all": "Close All",
+  "generic.refresh_all": "Refresh_All",
+  "generic.apply": "Apply",
+  "generic.mirror": "Mirror",
+  "generic.search_tags": "Search Tags",
+  "build_tags.parent_tags": "Parent Tags",
+  "build_tags.add_parent_tags": "Add Parent Tags",
+  "delete_unlinked.delete_unlinked": "Delete Unlinked Entries",
+  "delete_unlinked.confirm": "Are you sure you want to delete the following %{len(self.lib.missing_files)} entries?",
```

Comment:
@Bamowen if the underlying implementation changes (for example from variable `self.lib.missing_files` to `missing_files_count`), will it break the translation? If so, then that's quite brittle way how to make these translation string.

In such case it would be better to change the strings in code from f-string to use `format()` like this: 

```"Are you (...) {files_count} entries?".format(files_count=len(self.lib.missing_files))```, so the translation is not directly coupled to the variable name.

Reply:
I've been doing this in #447 as I go through changing the code

Reply:
nice